### ISSUE TITLE
Small error in setupInstance() documentation?

### ISSUE THREAD

**Issue by sebastien-prudhomme** [feathersjs-ecosystem/feathers-vuex#549]
I think there is this small error in setupInstance() documentation in ```docs/model-classes.md```:

```js
data.posts = data.posts.map(post => new models.Post(post))
```

should be replaces by:

```js
data.posts = data.posts.map(post => new models.api.Post(post))
```

Can someone confirm?

**Comment by hamiltoes** [comment]
It depends whether or not you've set up a `serverAlias` and if so what you've named that alias.

If you have set up a `serverAlias` of `api` then you will need to access the `Post` constructor at `models.api.Post` as you've indicated.

**Comment by sebastien-prudhomme** [comment]
The default value for serverAlias is ```api``` and all other code examples use ```models.api```.

I can read in the doc that ```Models are keyed by serverAlias```, this is not a mandatory field in the options?

**Comment by fratzinger** [comment]
I think @sebastien-prudhomme is right on this one. I think it's a leftover.
Could you do a pull request? Highly appreciated :)

**Comment by marshallswain** [comment]
Yep, those docs are outdated.  A PR would be appreciated.

**Comment by marshallswain** [comment]
Closed in #550
